source/api/presentation/3.0/index.md

```diff
@@ -541,7 +541,7 @@ The value _MUST_ be an array of JSON objects. Each object _MUST_ have the `id`,
 ```
 
 ##### rendering
-A link to an external resource that is an alternative, non-IIIF representation of this resource. The external resource _MUST_ be able to be displayed directly to a human user, and _MUST NOT_ have a splash page or other interstitial resource that mediates access to it. If access control is required, then the [IIIF Authentication API][auth-api] is _RECOMMENDED_. Examples might include the preferred viewing environment for the IIIF resource, such as a viewer page on the publisher's web site. Other uses include a rendering of a manifest as a PDF or EPUB with the images and text of the book, or a slide deck with images of the museum object.
+A link to an external resource that is an alternative, non-IIIF representation of this resource. Such representations typically cannot be painted onto a single Canvas, as they either include too many views (such as a PDF with multiple pages), have incompatible dimensions (such as a 3d model with a 2d Canvas), or are composite resources requiring additional rendering functionality. The external resource _MUST_ be able to be displayed directly to a human user, and _MUST NOT_ have a splash page or other interstitial resource that mediates access to it. If access control is required, then the [IIIF Authentication API][auth-api] is _RECOMMENDED_. Examples might include the preferred viewing environment for the IIIF resource, such as a viewer page on the publisher's web site. Other uses include a rendering of a book as a PDF or EPUB, a slide deck with images of a building, or a 3d model of a statue.  External resources that are related to this resource _MUST_ instead be referenced in the `metadata` field, or linked to a particular Canvas segment with an Annotation.
```

Comment:
> Examples might include the preferred viewing environment for the IIIF resource, such as a viewer page on the publisher's web site.

The trouble with that example is that it will often be the same page as pointed at by `homepage`. That doesn't mean it isn't a valid `rendering`, but how to avoid confusion? Should I point `homepage` and `rendering` at the same URI?

Reply:
True. No, it should just go in homepage. Will take out the example.